### ISSUE TITLE
[Screen Reader – Make Code – Project Setting] 'Go back' button is defined as heading<H3> present on 'Project setting' screen.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3687]
**User Experience:**
Properly defined information & relationship of an UI element helps people with different disabilities, by allowing user agents to provide proper navigation & content according to the needs of individual users.

**Note**: User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Project Settings" option and select it.
7. Navigate though all the controls appearing after select "Project Setting" option i.e. No Pairing Required: Anyone can connect via Bluetooth, Passkey pairing: Pairing requires 6 digit key to pair etc. 
8. Verify that  'Go back' button is defined as heading<H3> or not.

**Actual Result:**
'Go back' button is defined as heading<H3> present on 'Project setting' screen and screen reader is reading as 'Go back button heading level 3'.

**Expected Result:**
'Go back' button should not defined as heading<H3> present on 'Project setting' screen and screen reader should be read as 'Go back button'.

**Note:**
This issue also exists with Firefox and Chrome Browser.

**WCAG Reference:**
[WCAG1.3.1 - Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG1.3.1; #Win10-Edge(Chromium)

![WCAG1 3 1_Project Setting](https://user-images.githubusercontent.com/72871089/99033525-19569880-25a1-11eb-9569-c788d8836cbb.jpg)
